tests/core/test_evaluation.py

```diff
@@ -495,3 +495,86 @@ def test_log_evaluation_table(caplog, skip_field, skip_value):
         assert f"Classification report: \n{kwargs['report']}" in caplog.text
     else:
         assert "Classification report:" not in caplog.text
+
+
+@pytest.mark.parametrize(
+    "test_file, test_case",
+    [
+        [
+            "data/test_yaml_stories/test_prediction_with_correct_intent_wrong_entity.yml",
+            "correct_intent_wrong_entity",
+        ],
+        [
+            "data/test_yaml_stories/test_prediction_with_wrong_intent_correct_entity.yml",
+            "wrong_intent_correct_entity",
+        ],
+        [
+            "data/test_yaml_stories/test_prediction_with_wrong_intent_wrong_entity.yml",
+            "wrong_intent_wrong_entity",
```

Comment:
maybe rather than a string it would be easier to have 2 booleans e.g. `correct_intent` and `connect_entity`?